### ISSUE TITLE
Add left handed support

### ISSUE THREAD

**Issue by Odie** [Odie/skyui-vr#3]

**Comment by Odie** [comment]
It's not clear what this actually gets us. It's possible to review the SE => VR code changes again to see what the handedness code actually does. But as far as I can tell, both Vive wands are treated exactly the same in the UI.

If we ever actually grab events from OpenVR, input mapping likely needs to be redone anyway.

**Comment by BillyTPilgrim** [comment]
As a southpaw VR user, it's not really a functional issue, that works fine, but the UI displays the wrong hand for equipped weapons.

**Comment by Odie** [comment]
Okay! Thanks for letting me know. 

The vanilla game was crashing for me when I tried switching to left handed configuration some time ago. Maybe I should give it another go. Did you notice anything else other than the equip indicator/arrow being wrong? (So I can make a list of things to verify when this is worked on)

**Comment by Odie** [comment]
Is this what you’re referring to? https://www.nexusmods.com/skyrimspecialedition/mods/17729?tab=files

My guess is, the button prompt area might be missing VR icons. The item card “might” never fade back in after the first time.

Regarding integrating support for frostfall... that’d be a firm, “maybe?” :) I haven’t tried either survival or frostfall. Are we just looking at displaying an extra warmth value for the items and a warmth value next to the player health in the bottombar?

If that those are the only things different, it might be reasonable to just include it with the base install.

**Comment by BillyTPilgrim** [comment]
No, that's not it - Survival mode was something official added by Bethesda when the Creation Club was launched, and AFAIK that's never been active on VR. No, I'm talking about something different  - you remember when there was the hack for SkyUI going around r/skyrimvr where one copied and pasted some code into the decompiled .swf files? It's basically that fix, but applied to the .swf files from Frostfall. As far as I know the only difference is that the warmth and coverage values are shown on the item card for armours and clothing - I don't think there's even a warmth rating on the bottom bar.

The relevant files are here, if you want to take a look: https://drive.google.com/open?id=1wmCapLgsVkH1YmD8rK5budsI1GWHUi7G

Among other things it replaces containermenu, craftingmenu and inventorymenu which overwrite the default SkyUI ones, which is why I was saying your changes to those files would be wiped out.

**Comment by Odie** [comment]
> so A shows up as X and vice versa, and likewise B shows as Y and vice versa.
Yeah, I'm running a Vive, but having seen at least the basic layout of the Rift controller, this is perfectly clear. =)

I'll look into the handedness at some point. It really shouldn't be a lot of work, but I'm not sure when this might happen. I want to spend a bit of time to experiment and work out if we can get extra VR control/button data into the UI. If it's possible, it *might* change the way SkyUI deal with VR input entirely. Handedness will probably be deal with whatever input mapping system that drops into place.

See as things are *mostly* working. I'll mark this as low priority for now. =\

**Comment by BillyTPilgrim** [comment]
Understandable, functionality is the biggest priority. I'm not much of a coder, but I'll see if I can see the difference between the compiled vanilla SkyUI and the Frostfall version myself, and see if it's something that can just be dropped in.

I'm guessing you've seen this? https://www.reddit.com/r/skyrimvr/comments/9ls5vu/sksevrinputplugin_my_skse_plugin_that_provides_vr/

**Comment by Odie** [comment]
@BillyTPilgrim Let's move the Frostfall warmth discussion over to #15. =)

**Comment by Odie** [comment]
Things are moving forward at #9. Left handed input handling should eventually be handled by the todo item marked "VR input mapping layer". It's a bit silly to wait such a long time for the left handed handling. But what can ya do. I'm just one guy. =)
